web/auth/auth.go

```diff
@@ -0,0 +1,82 @@
+// Package auth provides register and login handlers
+package auth
+
+import (
+	"net/http"
+
+	"github.com/cozy/cozy-stack/apps"
+	"github.com/cozy/cozy-stack/web/middlewares"
+	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+)
+
+type registerForm struct {
+	Passphrase string `form:"passphrase"`
+	Token      string `form:"registerToken"`
```

Comment:
Unlike `encoding/json`, `gin/binding/form` does not specialcase []byte, it will be considered a slice and will not be properly decoded [(source)](https://github.com/gin-gonic/gin/blob/bb159f9260a262fccbdbd6f37f8575d1e7e6aaa0/binding/form_mapping.go#L45)